src/app.js

```diff
@@ -0,0 +1,65 @@
+const clicks = document.getElementById('clicks');
+const timer = document.getElementById('timer');
+const buttonClick = document.getElementById('buttonClick');
+const progress = document.getElementById('progress');
+const maxClicks = 5;
+let initialClick = 0;
+let time = 30;
+let timerId;
+
+const countClicks = () => initialClick += 1
+
+const discountTime = () => {
+  if (time === 0 && initialClick < maxClicks) {
+    perder();
+  }
+  time--;
+  timer.innerHTML = `00:${time}`;
+};
+
+const play = () => {
+  buttonClick.removeAttribute('disabled');
+  timerId = setInterval(discountTime, 1000)
+};
+
+const pause = () => {
+  buttonClick.setAttribute('disabled', true);
+  timerId = clearInterval(timerId)
```

Comment:
clearInterval no retorna un id
```suggestion
 clearInterval(timerId);
```

Reply:
listo